security-analysis/security-analysis-api/src/main/java/com/powsybl/security/results/PreContingencyResult.java

```diff
@@ -17,19 +18,24 @@ public class PreContingencyResult {
     private final LimitViolationsResult limitViolationsResult;
     private final NetworkResult networkResult;
 
+    private final LoadFlowResult.ComponentResult.Status mainComponentStatus;
```

Comment:
I'm not sure specifying "main component" is really necessary. It is also the case for all other results. Just "status" should be ok.